### ISSUE TITLE
Sync is broken

### ISSUE THREAD

**Issue by achingbrain** [ipfs-examples/js-ipfs-examples#44]
None of the examples have synced changes from this repo for a couple of weeks.

@hugomrdias @oliveriosousa any ideas?

**Comment by oliveriosousa** [comment]
You are right, there is a bug. The action is replacing all the content and removed the `workflow` folder. I will have a look.

**Comment by oliveriosousa** [comment]
In the meanwhile, I synced all the examples, should be good to test `js-ipfs`

**Comment by achingbrain** [comment]
This is still broken, I just tried to manually trigger the sync in `js-ipfs-types-use-ipfs-from-typed-js` and it fails with:

```
Merging changes...
HEAD is now at 7f37ce3 fix: add CI actions to examples (#45)
Pushing changes...
To ipfs-examples/js-ipfs-types-use-ipfs-from-typed-js
 ! [remote rejected] main -> main (refusing to allow a GitHub App to create or update workflow `.github/workflows/sync.yml` without `workflows` permission)
error: failed to push some refs to 'ipfs-examples/js-ipfs-types-use-ipfs-from-typed-js'
Error: Process completed with exit code 1.
```

https://github.com/ipfs-examples/js-ipfs-types-use-ipfs-from-typed-js/runs/3756008221

**Comment by oliveriosousa** [comment]
Latest run: https://github.com/ipfs-examples/js-ipfs-types-use-ipfs-from-typed-js/actions/runs/1291085593

**Comment by achingbrain** [comment]
Sync is broken again - some of the repos have this message on their actions page:

> This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days. Enable this workflow to resume scheduled runs.

This might be better as a push from https://github.com/ipfs-examples/js-ipfs-examples CI instead of polling the parent repo for changes.

**Comment by hugomrdias** [comment]
ah damn you probably right

**Comment by lidel** [comment]
@laurentsenta @galargh seems that CI that synchronizes https://github.com/ipfs-examples/js-ipfs-examples/tree/master/examples with standalone repos in this org got broken at some point,  would it be possible for IPDX  to help with this? (should I fill an issue somewhere, etc?)

**Comment by laurentsenta** [comment]
@lidel Thanks for the ping, it's on our board and I'll look into it,
in the future: if you add the label `team/ipdx` to an issue, it gets added to our board automatically during our triage sessions :)
